core/iwasm/interpreter/wasm_interp_classic.c

```diff
@@ -6545,6 +6545,13 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
             /* Initialize the local variables */
             memset(frame_lp + cur_func->param_cell_num, 0,
                    (uint32)(cur_func->local_cell_num * 4));
+            /* externref/funcref should be NULL_REF rather than 0 */
+            for (i = 0; i < cur_func->local_cell_num; i++) {
+                if (cur_wasm_func->local_types[i] == VALUE_TYPE_EXTERNREF
+                    || cur_wasm_func->local_types[i] == VALUE_TYPE_FUNCREF) {
+                    *(frame_lp + cur_func->param_cell_num + i) = NULL_REF;
```

Comment:
Is it only for reference types feature? Since for GC, the default NULL_REF is 0, memset 0 is also good. Had better add `#if WASM_ENABLE_REF_TYPES != 0 && WASM_ENABLE_GC == 0 ... #endif`?